cloudbuild.yaml

```diff
@@ -5,13 +5,13 @@ steps:
   - name: 'gcr.io/cloud-builders/docker'
     id: pull_image
     waitFor: ['-']
-    timeout: 10m
+    timeout: 30m
```

Comment:
Wait, did the image become **slower** to pull now?

Reply:
It timed out once, I presumed because of a cache miss that should go away.  (I still need to figure out why pushing to the cache doesn't work for me.)

Reply:
Docker cache? BTW, what happens if you run `./scripts/docker_push`, does it work on your machine?